### ISSUE TITLE
Duplicate-merging script has missed some duplicates.

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1722]
Duplicate-merging script has missed some duplicates.

https://tatoeba.org/eng/sentences/show/5753266
Tom is well connected.
https://tatoeba.org/eng/sentences/show/7414059
Tom is well connected.
edited by Hybrid, 13 days ago

https://tatoeba.org/eng/sentences/show/3780429
https://tatoeba.org/eng/sentences/show/7411458
Tom is strikingly handsome.
edited by Hybrid, 13 days ago

However, the script is apparently running.
https://tatoeba.org/eng/sentence_comments/of_user/Horus

7305143	eng	"What are you doing?" "I am repeating by myself."	jimkillock
7350148	eng	"What are you doing?" "I am repeating by myself."	jimkillock

There may be others.

**Comment by ckjpn** [comment]
This is still happening.
Here are a few examples found in this week's exported data.

https://tatoeba.org/eng/sentences/show/2541857
https://tatoeba.org/eng/sentences/show/7355027
I'm not a hundred percent sure.

https://tatoeba.org/eng/sentences/show/276955
https://tatoeba.org/eng/sentences/show/7496128
It's too expensive.

https://tatoeba.org/eng/sentences/show/242881
https://tatoeba.org/eng/sentences/show/7477236
It's hot today.

**Comment by ckjpn** [comment]
There is a Wall comment (2019-01-11) about this problem.

https://tatoeba.org/eng/wall/show_message/31119#message_31119

**Comment by jiru** [comment]
I ran the duplicate-merging script again over the whole corpus. The duplicates mentioned here and on the Wall have been deleted. In addition, I fixed what I think was causing the problem: I prevented the crontab from running multiple instances of the script at the same time by using `flock`, and I greatly speeded up the script execution time. If you see Horus missing duplicates again, feel free to open a new issue.
